src/JoomlaBrowser.php

```diff
@@ -240,25 +240,38 @@ public function setErrorReportingToDevelopment()
 	 */
```

Comment:
Please add the deprecated

```php
	 * ...
	 *
	 * @deprecated  since Joomla 3.4.4-dev. Use installExtensionFromFolder($path, $type = 'Extension') instead.
	 */
```